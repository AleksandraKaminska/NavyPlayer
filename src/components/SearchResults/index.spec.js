import React from 'react'
import { SearchResults } from './index'
import renderer from 'react-test-renderer'

describe('SearchResults', () => {
  it('should render self', () => {
    const rendered = renderer.create(<SearchResults />)
    expect(rendered.toJSON()).toMatchSnapshot()
  })
})
