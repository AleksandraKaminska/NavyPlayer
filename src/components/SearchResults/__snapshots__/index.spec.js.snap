// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResults should render self 1`] = `
<section
  className="search"
/>
`;
